src/SWP/Bundle/ContentBundle/Factory/ORM/MediaFactory.php

```diff
@@ -116,6 +101,44 @@ public function createImageRendition(
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function replaceBodyImagesWithMedia(ArticleInterface $article, ArticleMediaInterface $articleMedia)
```

Comment:
why is this method in the factory? it should be in some different service (ArticleBodyProcessor etc.) It doesn't fit here IMO.